declarative-config-bridge/src/main/java/io/opentelemetry/instrumentation/sdk/ConfigPropertiesUtil.java

```diff
@@ -3,7 +3,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package io.opentelemetry.instrumentation.api.incubator.sdk.config.bridge;
+package io.opentelemetry.instrumentation.sdk;
```

Comment:
let's update the package name too, maybe

```suggestion
package io.opentelemetry.instrumentation.config.bridge;
```